cmd/kube-apiserver/app/server.go

```diff
@@ -573,24 +573,24 @@ type completedServerRunOptions struct {
 // Complete set default ServerRunOptions.
 // Should be called after kube-apiserver flags parsed.
 func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
-	var options completedServerRunOptions
+	var cmpltdSvrOptions completedServerRunOptions
```

Comment:
Golang generally favors [shorter variable names](https://github.com/golang/go/wiki/CodeReviewComments#variable-names). It is true that the name as it is conflicts with the `options` package, but the `options` package isn't used from within this function, so I'm not so sure the name should be changed. Was there another reason you wanted to rename this variable?